lib/l10n/arb/app_en.arb

```diff
@@ -86,6 +86,7 @@
   "importSelectJsonFile": "Select JSON file",
   "importEnteEncGuide": "Select the encrypted JSON file exported from ente",
   "importRaivoGuide": "Use the \"Export OTPs to Zip archive\" option in Raivo's Settings.\n\nExtract the zip file and import the JSON file.",
+  "importBitwardenGuide": "Use the \"Export\" option in Bitwarden Settings.\n\nExtract the zip file and import the JSON file.",
```

Comment:
"Use the \"Export vault\" option within Bitwarden Tools and import the unencrypted JSON file.";

Reply:
ok